app/views/alaveteli_pro/account_request/index.html.erb

```diff
@@ -1,82 +1,135 @@
-        <li>A <strong class="marketing-highlight">searchable archive</strong> of Freedom of Information requests</li>
-        <li><strong class="marketing-highlight">Delivery verification</strong> for proof of receipt</strong></li>
-        <li>A permanent, searchable, <strong class="marketing-highlight">public record</strong> of your request and the responses</li>
-        <li><strong class="marketing-highlight">Streamlined process</strong> for requesting internal reviews</li>
+        <li>
+          <%=_('Up-to-date database of contact details for 20,000 ' \
+               'authorities') %>
+        </li>
+        <li>
+          <%=_('A searchable archive of Freedom of Information requests') %>
+        </li>
+        <li>
+          <%=_('Delivery verification for proof of receipt') %>
+        </li>
+        <li>
+          <%=_('A permanent, searchable, public record of your request and ' \
+               'the responses') %>
+        </li>
+        <li>
+          <%=_('Streamlined process for requesting internal reviews') %>
+        </li>
       </ul>
     </div>
   </div>
 </div>
 <div class="marketing__section">
   <div class="marketing__section__heading">
-    <h2>Coming soon - batch requests</h2>
+    <h2><%=_('Coming soon - batch requests') %></h2>
   </div>
   <div class="marketing__section__content">
-
-
     <div class="marketing__section__content-items">
       <ul class="marketing__section__feature-list">
-        <li><strong class="marketing-highlight">Make a request to multiple authorities</strong>: select authorities at the click of a button</li>
-        <li><strong class="marketing-highlight">Manage large volumes of responses</strong>:  easily keep track of the status of each request</li>
-        <li><strong class="marketing-highlight">Get regular updates</strong> as the responses come in — without overwhelming your inbox</li>
+        <li>
+          <%=_('Make a request to multiple authorities: select authorities at ' \
+               'the click of a button') %>
+        </li>
+        <li>
+          <%=_('Manage large volumes of responses:  easily keep track of the ' \
+               'status of each request') %>
+        </li>
+        <li>
+          <%=_('Get regular updates as the responses come in — without ' \
+               'overwhelming your inbox') %>
+        </li>
       </ul>
 
 
       <div class="side-by-side-elements screenshots">
         <div class="side-by-side-element">
-          <a href="<%= image_path("alaveteli-pro/screenshot-batch-selection.jpg") %>"><%= image_tag("alaveteli-pro/screenshot-batch-selection.jpg", :alt => "Batch requests allow multiple recipients of a single request") %></a>
+          <a href="<%= image_path("alaveteli-pro/screenshot-batch-selection.jpg") %>">
+            <%= image_tag("alaveteli-pro/screenshot-batch-selection.jpg",
+                          :alt => "Batch requests allow multiple recipients of a single request") %>
```

Comment:
We should probably make up the alt text for translation as well (we already do this in a few places)